apps/emqx_audit/src/emqx_audit.erl

```diff
@@ -67,7 +67,7 @@ to_audit(#{from := erlang_console, function := F, args := Args}) ->
         http_method = <<"">>,
         http_request = <<"">>,
         duration_ms = 0,
-        args = truncate_large_term({F, Args})
+        args = iolist_to_binary(io_lib:format("~p: ~ts", [F, Args]))
```

Comment:
Q: Does this mean we're back to potentially gigantic audit records of Erlang shell invocations?

Reply:
I guess it might be enough. If anyone is reporting this problem, we'll come back and fix it!